### PR TITLE
fix settings and random crashes

### PR DESCRIPTION
**Pull request by OH296**

### CODE REVIEW

File: scripts/scr_ui_settings/scr_ui_settings.gml
Comment (hristokn) [inline scripts/scr_ui_settings/scr_ui_settings.gml:1]: I don't undestand what this change does

**Review by hristokn — COMMENTED** [review]
I don't understand what the ui stuff does but the event script is ok

**Comment by OH296** [comment]
the ui stuff just makes sure the chapter settings screen always aligns properly

**Review by hristokn — APPROVED** [review]

**Review by hristokn — APPROVED** [review]

**Comment by hristokn** [comment]
oh I can't approve stuff, I think

**Pull request merged**